ccmlib/dse_node.py

```diff
@@ -64,12 +64,6 @@ def set_workloads(self, workloads):
         if 'solr' in self.workloads:
             self.__generate_server_xml()
         if 'graph' in self.workloads:
-            conf_file = os.path.join(self.get_path(), 'resources', 'dse', 'conf', 'dse.yaml')
-            with open(conf_file, 'r') as f:
-                data = yaml.load(f)
-            graph_options = data['graph']
-            graph_options['gremlin_server']['host'] = self.ip_addr
```

Comment:
Why do we need to remove more than just this line? It would seem like the rest of this code is still functional

Reply:
it looks like graph.gremlin_server.host is the only value changed, and with it gone, there's no need to read the yaml anymore?

Reply:
+1 to @xytxytxyt